src/VisualStudio/TestSetup/source.extension.vsixmanifest

```diff
@@ -17,7 +17,6 @@
   <Assets>
     <Asset Type="Microsoft.VisualStudio.MefComponent" Path="Roslyn.Hosting.Diagnostics.dll" />
     <Asset Type="Microsoft.VisualStudio.MefComponent" d:Source="Project" d:ProjectName="%CurrentProject%" Path="|%CurrentProject%|" />
-    <Asset Type="Microsoft.VisualStudio.VsPackage" Path="BindingPath.pkgdef" />
```

Comment:
This wasn't actually working since the file wasn't included in the VSIX. `ProvideCodeBase` is both more stable and more precise, so we'll use that instead.